draft-ietf-acme-acme.md

```diff
@@ -1624,7 +1647,9 @@ orders or authorization transactions based on a change of account key.
 
 A client can deactivate an account by posting a signed update to the server with
 a status field of "deactivated." Clients may wish to do this when the account
-key is compromised or decommissioned.
+key is compromised or decommissioned. A deactivated account can no longer request
+certificate issuance or access resources related to the account, such as orders
+or authorizations.
```

Comment:
Can you add one more sentence that explicitly says "If a server receives a POST or POST-as-GETs from a deactivated account it MUST return a urn:ietf:params:acme:error:unauthorized problem"?